src/lib.rs

```diff
@@ -91,17 +93,21 @@ pub trait Trace: Debug + Send {
 }
 
 /// The interface offered by all tracer types.
-///
-/// It is the job of the consumer of this library to decide which specific tracer type to
-/// instantiate, and to set it up using its type-specific configuration methods.
-pub trait Tracer {
+pub trait Tracer: Send + Sync {
```

Comment:
Do you need the explicit `Send` and `Sync`?

Reply:
No: they're auto-traits.

Reply:
Well yes, but this decision would move the error.

Without the annotation you would get an error near a spawn closure.

With you would get an error saying that the backend doesn't conform to our API.

Reply:
I don't get it. They trait is `Send + Sync` automatically (I think), so how can removing these change anything?

Reply:
A pub trait cannot be `Send + Sync` automatically, only it's implementers.

Reply:
That's the spirit!